skimage/transform/radon_transform.py

```diff
@@ -153,6 +153,11 @@ def iradon(radon_image, theta=None, output_size=None,
         Assume the reconstructed image is zero outside the inscribed circle.
         Also changes the default output_size to match the behaviour of
         ``radon`` called with ``circle=True``.
+    freqcutoff: float, optional (default 1, ie., no cutoff)
```

Comment:
@sciunto well this is not completely true :-). See for example ``transform.probabilistic_hough_line`` where default values are given just after the type. It's the same for ``transform.radon`` and ``transform.iradon``. I don't have strong feelings about this, but I think it can improve the readability of a docstring to put the default values on the same line as the parameter name.